doc/reference/tooling/tt_cli/configuration.rst

```diff
@@ -116,7 +131,13 @@ ee section
 ~~~~~~~~~~
 
 *   ``credential_path`` -- a path to the file with credentials used for
-    downloading Tarantool Enterprise Edition.
+    downloading Tarantool Enterprise Edition (Tarantool customer zone credentials).
+    The file should contain a username and a password, each on a separate line.
```

Comment:
Maybe, add a link to [Authentication](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/install/#authentication)? The example `cred.txt` file looks helpful.